conf.py

```diff
@@ -26,7 +26,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions += ['zeek', 'sphinx.ext.todo', 'zeek_pygments', 'literal-emph']
+extensions += ['zeek', 'sphinx.ext.todo', 'zeek_pygments', 'spicy-pygments', 'literal-emph']
```

Comment:
The `spicy-pygments` extension currently prevents parallel builds. I opened zeek/spicy#1530 to fix that.